test/swar/BasicOperations.cpp

```diff
@@ -54,7 +54,22 @@ static_assert(SWAR{Literals<4, u8>, {1, 2}}.value() == 0x12);
 static_assert(SWAR{Literals<16, u64>, {1, 2, 3, 4}}.at(0) == 4);
 static_assert(SWAR{Literals<16, u64>, {1, 2, 3, 4}}.at(1) == 3);
 
-static_assert(SWAR{Literals<4, u8>, {1, 2}}.to_array() == std::array<u8, 2>{1, 2});
+template <size_t N, typename A, typename B>
+constexpr auto compareContents(A a, B b) {
+    for (auto i = 0; i < N; ++i) {
+        if (a[i] != b[i]) {
+            return false;
+        }
+    }
+    return true;
+}
```

Comment:
`compareContents` of what? what's the comparison?
I know what you're trying to solv,e the lack of `constexpr` operator `==` for `std::array`.
Either solve that specifically, or, if better, you try to generalize, then really generalize.  I think your aim was toward equality, then by all means, implement the equality operator for things that resemble an `std::array`. 
This utility function is very needed.
`std::equal` almost does what is needed, but it has two design mistakes:
1. It does not check the size of the ranges
2. It wasn't `constexpr` from the beginning.
You may write `operator==`  for `T (&a)[Length]` and `std::array<T, Length>` using an implementation that is common to both.

This repository has a problem with half-thought code.

You don't need C++ 20 anymore with this utility, right?